.gitignore

```diff
@@ -43,6 +43,7 @@ terraform/.terraform/*
 .env
 .hypothesis/
 
+*.rst
```

Comment:
I don't think we want this in the final version?

Reply:
I agree with @aesharpe here -- we have many RST files which are checked into git and should be tracked.

Reply:
You might have forgotten @aesharpe but when you, Ella and I had a conversation about how about 150 rst files were pushed when I made my first changes on pudl, I suggested that maybe we could add it to the gitignore file. but since @zaneselvans has already confirmed that many .rst files still need to be tracked. I'll remove that and push the changes